openbb_terminal/stocks/stocks_helper.py

```diff
@@ -380,11 +379,13 @@ def load(
                 # Check that loading a stock was not successful
                 if df_stock_candidate.empty:
                     console.print("No data found.\n")
+                    return df_stock_candidate
+
             except Exception as e:
                 if "The API key provided is not valid" in str(e):
                     console.print("[red]Invalid API Key[/red]\n")
                 else:
-                    console.print(e)
+                    console.print(e, "\n")
```

Comment:
Does this print too many lines?  I always like just "" since rich seems to make a newline on print

Reply:
doneee